lib/membrane/rtp/twcc_receiver.ex

```diff
@@ -71,14 +74,20 @@ defmodule Membrane.RTP.TWCCReceiver do
   end
 
   @impl true
-  def handle_caps(Pad.ref(:input, ssrc), caps, _ctx, state) do
-    {{:ok, caps: {Pad.ref(:output, ssrc), caps}}, state}
+  def handle_caps(Pad.ref(:input, ssrc), caps, ctx, state) do
+    [caps: {Pad.ref(:output, ssrc), caps}] |> send_as_actions_or_buffer(ctx, state)
   end
 
   @impl true
-  def handle_event(Pad.ref(direction, ssrc), event, _ctx, state) do
+  def handle_pad_added(pad, _ctx, state) do
+    {actions, buffer} = Map.pop(state.actions_buffer, pad, [])
+    {{:ok, Enum.to_list(actions)}, %{state | actions_buffer: buffer}}
```

Comment:
Maybe we could rename `actions_buffer` to `buffered_actions`? When reading `buffer` in a callback, the  `%Membrane.Buffer{}` is always the first thing that comes to my mind 😅